makefile

```diff
@@ -0,0 +1,64 @@
+# MedGen ingest
```

Comment:
I just needed to change 1 path in order to get Chris' code to work. Pretty nice.